AGENTS.md

```diff
@@ -0,0 +1,243 @@
+# AGENTS.md - Development Guide for LLM Coding Agents
+
+This document provides comprehensive guidance for LLM coding agents working on the WeakAuras Companion project.
+
+## Project Overview
+
+WeakAuras Companion is a cross-platform desktop application built with Electron that serves as a bridge between [Wago.io](https://wago.io) and the WeakAuras World of Warcraft addon. It enables automatic fetching and updating of WeakAuras configurations without manual copy-paste operations.
+
+### Key Features
+- Automatic WeakAuras updates from Wago.io
+- Support for Plater profiles, mods, and scripts
+- Cross-platform support (Windows, macOS, Linux)
+- Automatic backup functionality
+- Multi-language support
```

Comment:
is this really a "key feature"? like yeah it does have i18n but that doesn't seem like it should receive equal billing as "downloads export strings from wago and puts an update button in /wa for you"